src/setupTests.js

```diff
@@ -0,0 +1,4 @@
+import Enzyme from 'enzyme';
```

Comment:
this file is also for React apps (not for this repo) - you could add a setupTestWriter to make this file as part of your script.